README.md

```diff
@@ -11,3 +11,24 @@ More importantly it's also an example web application written in the
 ```sh
 rebar3 shell
 ```
+
+## Docker
+
+Build with:
+
+```sh
+docker build -t gleam-echo:latest .
```

Comment:
Could we do this on CI to test?

Reply:
Might not be super high value if there's other stuff you'd rather do!

Reply:
Sounds like a good idea. I don't have any experience with github workflows so I'm content for someone else to do it. Given enough time I might try to explore it myself but certainly happy if someone knows the answer.

Reply:
Let's open an issue for it to be done later